requirements/edx/github.txt

```diff
@@ -74,7 +74,7 @@ git+https://github.com/edx/lettuce.git@0.2.20.002#egg=lettuce==0.2.20.002
 -e git+https://github.com/edx/event-tracking.git@0.2.1#egg=event-tracking==0.2.1
 -e git+https://github.com/edx/django-splash.git@v0.2#egg=django-splash==0.2
 -e git+https://github.com/edx/acid-block.git@e46f9cda8a03e121a00c7e347084d142d22ebfb7#egg=acid-xblock
-git+https://github.com/edx/edx-ora2.git@1.4.6#egg=ora2==1.4.6
+git+https://github.com/edx-olive-oc/edx-ora2.git@a0a8128febd06b0befff4899f070baa2e0504f14#egg=ora2==1.4.6.1
```

Comment:
@itsjeyd Is it ok to use the branch name `version_1.4.6` here instead of the commit hash?  That way we don't have to submit a version bump PR every time we merge something to ORA2.

Reply:
@pomegranited Sure, yeah -- that's a good point :)

@diegomillan Could you go ahead and make that update? Then we can merge this PR. (Jill and I discussed the test failures offline: As she mentioned [above](https://github.com/edx-olive-oc/edx-platform/pull/14#issuecomment-432902569), the errors reported are consistent with an issue that we've seen in other contexts. And it hasn't been that long since we got CI tests to work at all on the `edx-olive-oc/edx-platform` fork, so it should be fine to merge this PR without a green build, and address the test failures via a separate ticket.)

Reply:
Hi @pomegranited @itsjeyd, I just changed the requirement so it can be installed directly from `version_1.4.6`. But I have one question: suppose we merge another feature into `version_1.4.6` branch, wouldn't we have to also bump the internal package version for the current virtualenv???For example, in this case from 1.4.6.1 to 1.4.6.X, that way pip can detect and install any update for the package.

Reply:
@diegomillan Sorry for the delay here, I accidentally lost track of your comment.

> ... wouldn't we have to also bump the internal package version for the current virtualenv???For example, in this case from 1.4.6.1 to 1.4.6.X, that way pip can detect and install any update for the package.

I don't think so: Paver uses [this `pip` command](https://github.com/edx-olive-oc/edx-platform/blob/fe1bbebd926e5534180986cea1e51abffded38ea/pavelib/prereqs.py#L161) when installing prereqs which includes `--exists-action w`. So if there is an existing install of `edx-ora2`, it should get overwritten  regardless of whether the requirements file specifies a Git tag, hash, or branch. Cf. relevant parts of [`pip` documentation](https://pip.pypa.io/en/latest/reference/pip/#exists-action-option):

> This option specifies default behavior when path already exists. Possible cases: downloading files or checking out repositories for installation, creating archives. If `--exists-action` is not defined, pip will prompt when decision is needed.
> 
> ...
> *(w)ipe*
>     Delete the file or VCS checkout before trying to create, download, or checkout a new one.
> ...

Based on this, it should be fine to just keep `1.4.6` for the `VERSION` part of the GitHub reference, i.e.:

```
git+https://github.com/edx-olive-oc/edx-ora2.git@version_1.4.6#egg=ora2==1.4.6
```